### ISSUE TITLE
Less tests are executed after #3427 

### ISSUE THREAD

**Issue by mdelapenya** [elastic/apm-server#3436]
While debugging another issue, @v1v has noticed that there is a set of tests that has been removed from the CI execution when https://github.com/elastic/apm-server/pull/3427 was merged:
- [build](https://apm-ci.elastic.co/job/apm-server/job/apm-server-mbp/job/master/372) that ran before the above PR was merged.
- [build](https://apm-ci.elastic.co/job/apm-server/job/apm-server-mbp/job/master/373) that ran after the above PR was merged.

For instance:

![image](https://user-images.githubusercontent.com/2871786/76070241-1922e980-5f8c-11ea-9b1c-3c3e301c876c.png)

vs

![image](https://user-images.githubusercontent.com/2871786/76070264-2213bb00-5f8c-11ea-982d-2c1bc6044046.png)

There are 159 tests that have been skipped:

![image](https://user-images.githubusercontent.com/2871786/76070304-348df480-5f8c-11ea-9232-b552d12b71fc.png)

We'd like to make those tests run again
